src/crimson/os/seastore/lba_manager/btree/btree_lba_manager.cc

```diff
@@ -633,6 +635,51 @@ BtreeLBAManager::update_mapping(
   );
 }
 
+BtreeLBAManager::update_mappings_ret
+BtreeLBAManager::update_mappings(
```

Comment:
I think the change is good. My only question is why not reusing `LBAManager::update_mapping()`?

Reply:
That requires `update_mapping()` to accept a freestyle nullable ptr like before. But this is a special case which should only used by `LBAManager::update_mappings()` --- it can be hidden inside the lba implementations.

I was trying to make `LBAManager` public interfaces more strict about when the user should pass extents, so we don't need to search the entire code base to understand the possible scenarios.

Reply:
The scenario of `update_mappings(extents)` in TM is different from `update_mapping(extent)` because it assumes the extents are already linked.